src/routes/files.test.js

```diff
@@ -149,10 +149,9 @@ describe('Forms route', () => {
         }
       })
 
-      expect(response.statusCode).toEqual(StatusCodes.BAD_REQUEST)
+      expect(response.statusCode).toEqual(StatusCodes.OK)
       expect(response.result).toMatchObject({
-        error: 'Bad Request',
-        message: '"form.file.fileStatus" must be [complete]'
+        message: 'Ingestion failed'
```

Comment:
From the logs, will we have a way to see _why_ the validation failed?

Probably should match the text to what we send to `logger.info()`

```suggestion
        message: 'Ingestion failed validation'
```

Reply:
Yes - [we write](https://github.com/DEFRA/forms-submission-api/blob/f37836dc74df2d28ccc8a478223442e0fa7a224a/src/routes/files.js#L45) the `err` with the log.

I was more trying to match the message returned in the [main handler](https://github.com/DEFRA/forms-submission-api/blob/f37836dc74df2d28ccc8a478223442e0fa7a224a/src/routes/files.js#L30).

To be honest - they (CDP) are not going to look at what's returned, I just wanted something so it wasn't an empty response.

Reply:
Brill, just checking we've still got it if needed

Thanks for matching them up